### ISSUE TITLE
Cannot Run Examples

### ISSUE THREAD

**Issue by jd-carroll** [dmvaldman/samsara#48]
Seeing the following error in the logs:
_MergedStream.js:37 Uncaught TypeError: stream.on is not a function

**Comment by jd-carroll** [comment]
Stacktrace:
```
ReduceStream (ReduceStream.js:9)
initialize (SequentialLayout.js:45)
Controller (Controller.js:71)
View (View.js:83)
(anonymous function) (Controller.js:145)
child (Controller.js:146)
initialize (Dots.js:24)
Controller (Controller.js:71)
View (View.js:83)
(anonymous function) (Controller.js:145)
child (Controller.js:146)
createDots (Carousel.js:88)
initialize (Carousel.js:19)
Controller (Controller.js:71)
View (View.js:83)
(anonymous function) (Controller.js:145)
child (Controller.js:146)
(anonymous function) (main.js:9)
execCb (require.js:1693)
...
```

**Comment by joeandrews** [comment]
Last working commit is `cd013d8a38fdb7db187a2bda6d71b983bf9bfe1f`

**Comment by jd-carroll** [comment]
Looks like the commit# e5f18ea8731a9924df750a5b7275daecda7c537a broke it with [this line](https://github.com/dmvaldman/samsara/commit/e5f18ea8731a9924df750a5b7275daecda7c537a#diff-8ae1af4b0a91ffb4ce477ddd50193137R48)

**Comment by jd-carroll** [comment]
The object needs to be a stream object, and I am new here so I don't know what the original issue was which required this change.

**Comment by jd-carroll** [comment]
So in commit# e5f18ea there are 2 separate changes.  One is to `ui/Slider.js` and the other is to `layouts/SequentialLayout.js`.  The change to `SequentialLayout.js` is not related to the commit message.

**Comment by jd-carroll** [comment]
Reverting the change at e5f18ea to `layouts/SequentialLayout.js` fixes the issue...

But should understand why that change was included in the first place.

**Comment by dmvaldman** [comment]
Thanks for pointing this out! Sorry it took me a few days to respond. I'll be looking into the soon. Should be a simple fix.

**Comment by dmvaldman** [comment]
This should be resolved in b09b6bfc97b1ac28d4b18049ace3317c10eebc3c, would appreciate if you could verify.

**Comment by dmvaldman** [comment]
Closing as this has been fixed.
